### PR TITLE
Add DEFAULT DATABASE and USER MANAGEMENT to the database administration section.

### PR DESCRIPTION
**Pull request by Lojjs**

### CODE REVIEW

**Review by Hunterness — APPROVED** [review]
The database privilege image is also updated to reflect this and other 4.1 changes: https://github.com/neo-technology/neo4j-manual-modeling/pull/1030

File: cypher/cypher-docs/src/docs/dev/ql/administration/security/database/admin-database-syntax.asciidoc [hunk 1]
```diff
@@ -5,68 +5,68 @@
 
 | [source, cypher]
 GRANT ACCESS
-    ON DATABASE[S] {name \| *}
+    ON {DEFAULT DATABASE \| DATABASE[S] {name \| *}}
     TO role[, ...]
-| Allow the specified role or roles to access the database `name` or all databases
+| Allow the specified role or roles to access the default database, the database `name` or all databases
```
Comment (Hunterness) [inline cypher/cypher-docs/src/docs/dev/ql/administration/security/database/admin-database-syntax.asciidoc:10]: There are individual files for some of the different privileges, maybe they should be updated too?

```
database/all-management-syntax.asciidoc
database/constraint-management-syntax.asciidoc
database/deny-database-access-syntax.asciidoc
database/deny-database-start-syntax.asciidoc
database/deny-database-stop-syntax.asciidoc
database/grant-database-access-syntax.asciidoc
database/grant-database-start-syntax.asciidoc
database/grant-database-stop-syntax.asciidoc
database/index-management-syntax.asciidoc
database/name-management-syntax.asciidoc
```

**Review by Hunterness — CHANGES_REQUESTED** [review]
When updating docs for the compound work I found more places where this maybe should have been updated

**Review by Hunterness — APPROVED** [review]

**Review by davidoliverSP2 — APPROVED** [review]
Thanks @Lojjs 👍 

The only feedback I have is that I noticed on quite a few occasions the user of dbms in lower case - I think it should be consistent with [the introduction to databases and graphs](https://neo4j.com/docs/cypher-manual/4.1-preview/introduction/neo4j-databases-graphs/), and be in upper case. However, I don't think it's worth changing in this PR, as I saw multiple instances of it in the manual - we'll fix in another PR.

**Pull request merged**